scripts.d/50-libaribcaption.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+SCRIPT_REPO="https://github.com/xqq/libaribcaption"
+SCRIPT_COMMIT="0a3a209d3f0b3650b92941248b9701bf69584a1f"
+
+ffbuild_enabled() {
+    # since 6.1
+    [[ $ADDINS_STR == *4.4* ]] && return -1
+    [[ $ADDINS_STR == *5.0* ]] && return -1
+    [[ $ADDINS_STR == *5.1* ]] && return -1
+    [[ $ADDINS_STR == *6.0* ]] && return -1
+
+    return 0
+}
+ffbuild_dockerdl() {
+    default_dl "$SELF"
+}
+ffbuild_dockerbuild() {
+    mkdir -p "$FFBUILD_DLDIR/$SELF"/build
+    cd "$FFBUILD_DLDIR/$SELF"/build
+    cmake .. -DCMAKE_BUILD_TYPE=Release -DARIBCC_USE_FREETYPE:BOOL=ON -DCMAKE_TOOLCHAIN_FILE="$FFBUILD_CMAKE_TOOLCHAIN" -DCMAKE_INSTALL_PREFIX="$FFBUILD_PREFIX"
+    make -j$(nproc)
+    make install
+    if [[ $TARGET == linux* ]]; then
```

Comment:
since the generated pc file for windows build links `/opt/ct-ng/x86_64-w64-mingw32/lib/libstdc++.a` like below
```
cat /opt/ffbuild/lib/pkgconfig/libaribcaption.pc
prefix=/opt/ffbuild
exec_prefix=${prefix}
libdir=/opt/ffbuild/lib
includedir=/opt/ffbuild/include

Name: libaribcaption
Description: Portable ARIB STD-B24 Caption Decoder/Renderer
URL: https://github.com/xqq/libaribcaption
Version: 1.1.1
Requires: freetype2
Requires.private:
Cflags: -I${includedir}
Libs: -L${libdir} -laribcaption -l/opt/ct-ng/x86_64-w64-mingw32/lib/libstdc++.a -lssp_nonshared -lssp -lfreetype -lole32 -ld2d1 -ldwrite -lwindowscodecs
Libs.private:
Libs.private: -lstdc++
^^^
remove to avoid pkg-config error
```